### DISCUSSION TITLE
Unable to download Foundry on Windows

### DISCUSSION THREAD

**Discussion by SumiTechTo** [Cyfrin/security-and-auditing-full-course-s23#99] [Q&A]
This is what my Bash command line looks like:
Sumita Pathak@LAPTOP-52PGEQL9 MINGW64 ~ (mem)
$ curl -L https://foundry.paradigm.xyz | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0Installing foundryup...

100  1942  100  1942    0     0   3863      0 --:--:-- --:--:-- --:--:--  3863
mkdir: cannot create directory ‘/c/Users/Sumita’: File exists

Please let me know how can I fix it

**Comment by PatrickAlphaC** [comment]
Are you using Windows WSL? Foundry doesn't work great on raw windows. 

During the installation part of the course, we walk through getting windows WSL setup

**Comment by SumiTechTo** [comment]
Hey thanks for the reply. I am thinking of completely shifting to Linux,
for this course as well as for my future opportunities

**Reply by xaiksan1** [reply]
1st your mkdir don't work cause your file already exists. If you use VSC instead of shifting to Linux;you can install this 3 extensions; WSL, VSCode Terminal for Ubuntu+ Open(Wsl support). Be sure to have WSL 2 installed properly in windows. WINKEX could be a great addon too cause it allows you to use linux natively under windows and from recently even with cluster. Git Bash could work too, but it still a lightweight solution and maybe further in the course it will not be enough.

**Comment by pymachado** [comment]
Hi @SumiTechTo, I'm using Windows, and to install Foundry, I had to install [Git Bash](https://git-scm.com/download/win) as a lightweight solution. To this day, it is working perfectly. So, before shifting to Linux, try installing Git Bash, and after that, install Foundry using this console.

**Comment by SumiTechTo** [comment]
Hey thanks for the reply. I have already installed Git Bash but facing a
server error as examined by a senior.

**Comment by EngrPips** [comment]
@SumiTechTo  did you resolve the server error already ?

**Comment by panditdhamdhere** [comment]
> This is what my Bash command line looks like:
> Sumita Pathak@LAPTOP-52PGEQL9 MINGW64 ~ (mem)
> $ curl -L https://foundry.paradigm.xyz | bash
>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
>                                  Dload  Upload   Total   Spent    Left  Speed
>   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0Installing foundryup...
> 
> 100  1942  100  1942    0     0   3863      0 --:--:-- --:--:-- --:--:--  3863
> mkdir: cannot create directory ‘/c/Users/Sumita’: File exists
> 
> Please let me know how can I fix it

I was facing same problem but I solved it.

**Comment by SumiTechTo** [comment]
Hey thank you so much. I think this will be really helpful.
